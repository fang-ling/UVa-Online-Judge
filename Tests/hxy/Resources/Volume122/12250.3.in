VALAASONI
OMAIGO
HELLO
ZDRAVSTVUJTE
BUGBUG
HALLO
UDEBUG
HOLA
CIAO
BANGLA
BONJOUR
OIHUN
HOLAA
ZDRAVSTVUJTE
ZOBBOR
#
