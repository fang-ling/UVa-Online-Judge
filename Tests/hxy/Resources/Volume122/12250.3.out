Case 1: UNKNOWN
Case 2: UNKNOWN
Case 3: ENGLISH
Case 4: RUSSIAN
Case 5: UNKNOWN
Case 6: GERMAN
Case 7: UNKNOWN
Case 8: SPANISH
Case 9: ITALIAN
Case 10: UNKNOWN
Case 11: FRENCH
Case 12: UNKNOWN
Case 13: UNKNOWN
Case 14: RUSSIAN
Case 15: UNKNOWN
